ext/zend_test/test.stub.php

```diff
@@ -150,6 +150,12 @@ class Foo {
         public function method(): int {}
     }
 
+    class UnlikelyCompileError {
+        /* This method signature would create a compile error due to the string
+         * "ZendTestNS\UnlikelyCompileError" in the generated macro call */
+        public function method(): UnlikelyCompileError {}
```

Comment:
```suggestion
        public function method(): UnlikelyCompileError { return $this; }
```

maybe?

Reply:
The stub code is irrelevant, as it's only used to generate the arginfo structure, not define the actual behaviour of the class. I was lazy and didn't implement the logic in zend_test.c, but will circle back later to do so.

Reply:
However currently it causes an error when running tests, e.g. https://github.com/php/php-src/runs/7521368881?check_suite_focus=true#step:9:139

Reply:
I believe this is due to the implementation of the class (which is in zend_test.c), but that's the error I was going to take care of later.

Reply:
No, that won't help (and may even break gen_stub.php).

Reply:
Sorry I was confused (between PHP stub and C impl), now I see other methods above all have an empty body; please disregard my suggestion